index.js

```diff
@@ -1,11 +1,44 @@
 'use strict';
 const fs = require('fs');
+const os = require('os');
 const path = require('path');
 const childProcess = require('child_process');
 const runApplescript = require('run-applescript');
 const pify = require('pify');
 const rimraf = require('rimraf');
-const xdgTrashdir = require('xdg-trashdir');
```

Comment:
Looks like you could pretty much reuse most of this logic https://github.com/kevva/xdg-trashdir/blob/345387560a38dafb5aaddadba6cbfa7d2b3620a6/index.js#L31-L50